src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx

```diff
@@ -0,0 +1,35 @@
+---
+page_id: 6bf993fc-a195-4836-8eaf-133812be8876
+title: About scopes and Kinde API access
+sidebar:
+  order: 3
+relatedArticles:
+  - 51899f7f-3436-46e0-9a1b-6ecc3603a0df
+  - 50284476-2442-414c-af20-01ed3ef4ca4e
+  - 601dd8c5-6ee1-474f-ad36-201e65280462
+app_context:
+  - m: settings
+    s: applications
+  - m: user
+    s: apis
+---
+
+When you set up an M2M connection to Kinde's Management API, you also need to include the scopes that define API permissions.
```

Comment:
The Kinde management API uses JSON Web Tokens (JWTs) to authenticate requests. The token’s `scopes` claim indicates which endpoints can be accessed when calling the management API.

## Example scopes

- `read:users` for fetching user details.
- `update:users` for modifying user details.
- `read:roles` or `update:roles` for managing roles.

You can select scopes after you authorize a new M2M application or you can add scopes to an existing M2M application. We recommend adding as few scopes as you need, to maintain API security.